vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/ResourceCheckSteps.java

```diff
@@ -119,24 +119,38 @@ private ResourceValidation validate(ResourceValidation r)
         return HtmlUtils.getElements(html, cssSelector).parallelStream();
     }
 
+    private boolean isSourceAttributeEmpty(Element element)
+    {
+        boolean isSourceAttributeEmpty = getElementSourceAttribute(element).isEmpty();
```

Comment:
isSourceAttributeEmpty -> sourceAttributeEmpty

Reply:
why do you think so? https://rules.sonarsource.com/java/tag/convention/RSPEC-2047

Reply:
My comment is related to local var, not to method name